### DISCUSSION TITLE
Error 403 on a page

### DISCUSSION THREAD

**Discussion by Darwin1998** [spatie/laravel-permission#2217] [Q&A]
hello i am facing an issue where i already declared a permission for my admin role to view certain pages but there's this one page where i am always getting an error 403, the permission middleware is already registered in the kernel.php

**Comment by erikn69** [comment]
You must share a example app and more info

**Comment by drbyte** [comment]
It's possible that you've got a route definition that's incorrect/incomplete, or problems in your model, or a conflict of guards where it's checking against one you weren't expecting.

It will be helpful if you post at least enough code to demonstrate the problem. Preferably enough code to recreate the problem in a standalone app.
